wren-ui/src/apollo/server/mdl/mdlBuilder.ts

```diff
@@ -113,14 +113,27 @@ export class MDLBuilder implements IMDLBuilder {
     this.manifest.views = this.views.map((view: View) => {
       // if putting properties not string, it will throw error
       // filter out properties that have string value
-      const properties = pickBy<ViewMDL['properties']>(
-        JSON.parse(view.properties),
-        (value) => typeof value === 'string',
-      );
+      const properties = JSON.parse(view.properties) || {};
```

Comment:
The line `filter out properties that have string value` seems can be remove.

Reply:
addressed in da28c69